### DISCUSSION TITLE
Inquiry

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#38] [Q&A] [answered]
### What happened?

Is it possible to introduce different death animations of different monsters/enemies in the /kill function? and also is it better to use the triggermod to trigger "the lure enemies" function or do i just use /hate.

### Current Commit Hash

7f9e9df

### Injector

- [ ] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have **recursively** cloned the repository as instructed in the [README](https://github.com/winstxnhdw/lc-hax/edit/main/README.md)
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment, marked as answer]
Performing different kill animations is possible but specific conditions have to be met. For example, if you want to kill someone with the Forest Giant animation, a Forest Giant must exist on the map, and because of these specific conditions, we do not think that this feature is worth implementing.

TriggerMod is just a quality-of-life extension for `/hate` and several other commands. It is up to preference. Using `/hate` directly has the added benefit that it tells you if no enemies are found.

**Comment by yujispin990** [comment]
alrighty my last question, i've used an enhancement that  implemented "edit quota" without needing to be the host, i set it to 10000 and the moment we touched down in the company building i saw it change visually but i do not know if it is client sided or not , could i get your opinion on this?

**Comment by yujispin990** [comment]
ofc if you think this is a host only feature you can ignore the comment and just close thank you <3

**Comment by yujispin990** [comment]
ah just saw the person who made the enhancement comment that it is a host only feature and needs the lobby to be reloaded. I have gotten my answer, Thank you for the reply i really appreciate it, i will close now.
